src/admin/menu/class-bem-menu-walker.php

```diff
@@ -76,7 +78,7 @@ public function display_element( $element, &$children_elements, $max_depth, $dep
    *
    * @since 1.0.0
    */
-  public function start_lvl( &$output, $depth = 1, $args = array() ) { // phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.FoundInExtendedClassAfterLastUsed
+  public function start_lvl( &$output, $depth = 1, $args = [] ) { // phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.FoundInExtendedClassAfterLastUsed
```

Comment:
The comment should be added to the `phpcs.xml.dist`

```xml
<rule ref="Generic.CodeAnalysis.UnusedFunctionParameter">
  <severity>0</severity>
</rule>
```
Should silence all these errors.

Reply:
👍 tnx